docs/source/notebooks/Image_Tut.ipynb

```diff
@@ -0,0 +1,468 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Image Classification with Cleanlab, PyTorch & Skorch"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This tutorial will use Cleanlab to find potential label errors in the MNIST handwritten digit image dataset. This dataset contains 60,000 images of handwritten digits from 0 to 9. We'll find *more than a thousand* highly suspicious examples with label errors and edge cases by the end of this tutorial. \n",
```

Comment:
"This tutorial will use Cleanlab to find potential label errors in the MNIST handwritten digit image dataset"
->
"This XXmin cleanlab tutorial demonstrates how to find potential label errors in the MNIST handwritten digit image dataset"

Here: XX = estimated time to complete tutorial, eg. 5min

Reply:
done